parsons/zoom/zoom.py

```diff
@@ -0,0 +1,156 @@
+        self.client.data_key = data_key
+        data = self.client.data_parse(r)
+
+        if not params:
+            params = {}
+
+        # Return a dict or table if only one item.
+        if 'page_number' not in r.keys():
+            if isinstance(data, dict):
+                return data
+            if isinstance(data, list):
+                return Table(data)
+
+        # Else iterate through the pages and return a Table
+        else:
+            while r['page_number'] < r['page_count']:
+                params['page_number'] = int(r['page_number']) + 1
+                r = self.client.get_request(ZOOM_URI + endpoint, params=params, **kwargs)
+                data.extend(self.client.data_parse(r))
+            return Table(data)
+
+    def get_users(self, status='active', role_id=None):
+        """
+        Get users.
+
+        `Args:`
+            status: str
+                Filter by the user status. Must be one of following: ``active``,
+                ``inactive``, or ``pending``.
+            role_id: str
+                Filter by the user role.
+        `Returns:`
+            Parsons Table
+                See :ref:`parsons-table` for output options.
+        """
+
+        if status not in ['active', 'inactive', 'pending']:
+            raise ValueError('Invalid status type provided.')
+
+        params = {'status': status,
+                  'role_id': role_id}
+
+        tbl = self.get_request('users', 'users', params=params)
+        logger.info(f'Retrieved {tbl.num_rows} users.')
+        return tbl
+
+    def get_meetings(self, user_id, meeting_type='scheduled'):
+        """
+        Get meetings scheduled by a user.
+
+        `Args:`
+            user_id: str
+                A user id or email address of the meeting host.
+            meeting_type: str
+
+                .. list-table::
+                    :widths: 25 50
+                    :header-rows: 1
+
+                    * - Type
+                      - Notes
+                    * - ``scheduled``
+                      - This includes all valid past meetings, live meetings and upcoming
+                        scheduled meetings. It is the equivalent to the combined list of
+                        "Previous Meetings" and "Upcoming Meetings" displayed in the user's
+                        Meetings page.
+                    * - ``live``
+                      - All the ongoing meetings.
+                    * - ``upcoming``
+                      - All upcoming meetings including live meetings.
+        `Returns:`
+            Parsons Table
+                See :ref:`parsons-table` for output options.
+        """
+
+        tbl = self.get_request(f'users/{user_id}/meetings', 'meetings')
+        logger.info(f'Retrieved {tbl.num_rows} meetings.')
+        return tbl
+
+    def get_past_meeting(self, meeting_uuid):
+        """
+        Get metadata regarding a past meeting.
+
+        `Args:`
+            meeting_uuid: str
```

Comment:
In the name of consistency with the args for `get_past_meeting_participants`, this could be renamed to just `meeting_id`.